src/Microsoft.Health.Dicom.Tests.Common/Comparers/DicomFileEqualityComparer.cs

```diff
@@ -5,31 +5,45 @@
 
 using System.Collections.Generic;
 using Dicom;
+using EnsureThat;
 
 namespace Microsoft.Health.Dicom.Tests.Common.Comparers
 {
     public class DicomFileEqualityComparer : IEqualityComparer<DicomFile>
     {
+        public DicomFileEqualityComparer()
+           : this(new DicomTag[0])
+        {
+        }
+
+        public DicomFileEqualityComparer(IEnumerable<DicomTag> ignoredTags)
+        {
+            EnsureArg.IsNotNull(ignoredTags, nameof(ignoredTags));
+            IgnoredTags = ignoredTags;
+        }
+
         bool IEqualityComparer<DicomFile>.Equals(DicomFile x, DicomFile y)
```

Comment:
Is there a particular reason we're explicitly implementing?

Reply:
no. will fix it.